packages/venia-concept/src/RootComponents/Product/Product.js

```diff
@@ -60,6 +24,15 @@ class Product extends Component {
         window.scrollTo(0, 0);
     }
 
+    // map Magento 2.3.1 schema changes to v2.0.0 proptype shape to maintain backwards compatibility
```

Comment:
Could make this comment a little more clear that "v2.0.0" refers to PWA and not Magento.

Edit: in all the places where this comment appears

Reply:
The `v` stands for Venia, come on 😁 (updated four occurrences of this comment)